src/com/cdd/bao/template/ClipboardSchema.java

```diff
@@ -293,6 +291,7 @@ else if (annot.literal != null)
 			if (obj.optBoolean("exclude", false)) val.spec = Schema.Specify.EXCLUDE;
 			else if (obj.optBoolean("wholeBranch", false)) val.spec = Schema.Specify.WHOLEBRANCH;
 			else if (obj.optBoolean("excludeBranch", false)) val.spec = Schema.Specify.EXCLUDEBRANCH;
+			else if (obj.optBoolean("container", false)) val.spec = Schema.Specify.CONTAINER;
```

Comment:
I'm wondering if this is the right place for this code. Or you could convert this into a loop over the values of Specify:
```
for (Specify spec : Schema.Specify.values()) {
  String key = spec.getJsonKey();
  if (!key.equals("item") && obj.optBoolean(key, false)) {
    val.spec = spec;
    break;
  }
}
```

Reply:
Will revisit.  I just made the most obvious addition.